ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_isis.schema.yml

```diff
@@ -56,13 +56,21 @@ keys:
             - "ipv4 unicast"
             - "ipv6 unicast"
           description: Address Family
+          deprecation:
+            warning: true
+            new_key: address_family_ipv4/address_family_ipv6
```

Comment:
Same elsewhere 
```suggestion
            new_key: address_family_ipv4.enabled or address_family_ipv6.enabled
```

Reply:
We should add a new `enabled` key under each address family, so we never have a situation where we can have a "dangling" key. The issue is in YAML, if you just want to have the address family activated but no specific config below, you would write:

```yaml
router_isis:
  address_family_ipv4:
```

which YAML interprets as `address_family_ipv4: null` so when we do our `arista.avd.defined` check, it fails.

I know that requiring this `enabled` flag is breaking, but now is the time for breaking changes :)